vibi-dpu/src/utils/repo_config.rs

```diff
@@ -0,0 +1,26 @@
+use serde::{Serialize, Deserialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct RepoConfig {
+    comment: bool,
+    auto_assign: bool
+}
+
+impl RepoConfig {
+    // Getters
+    pub fn comment(&self) -> bool {
+        self.comment
+    }
+
+    pub fn auto_assign(&self) -> bool {
+        self.auto_assign
+    }
+
+    // Function to create a default RepoConfig
+    pub fn default() -> Self {
+        RepoConfig {
+            comment: false,
+            auto_assign: false
```

Comment:
Why default to false?
I think we should default to true.

(high LOGAF)

Reply:
Done